model-api/src/main/java/edu/unc/lib/boxc/model/api/ResourceType.java

```diff
@@ -28,29 +28,23 @@
  *
  */
 public enum ResourceType {
-    AdminUnit(1, Cdr.AdminUnit),
```

Comment:
These sort orders are still needed, they are used for sorting by object type and are associated with the RESOURCE_TYPE_SORT solr field